### PR TITLE
Homework7

### PR DESCRIPTION
**Pull request by OlegShvets1**
Homework7

### CODE REVIEW

File: src/Lesson_7/Task_2/FahrenheitConverter.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package Lesson_7.Task_2;
+
+public class FahrenheitConverter implements Converter {
+    @Override
+    public void convert(double celsius) {
+        double Fahrenheit;
```
Comment (Evangard) [inline src/Lesson_7/Task_2/FahrenheitConverter.java:6]: Naming convention.

File: src/Lesson_7/Task_2/KelvinConverter.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package Lesson_7.Task_2;
+
+public class KelvinConverter implements Converter{
+    @Override
+    public void convert(double celsius) {
+       double Kelvin;
```
Comment (Evangard) [inline src/Lesson_7/Task_2/KelvinConverter.java:6]: Naming convention.

File: src/Lesson_7/Task_3/MyDevice.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+            System.out.println("The music plays softly");
+        }
+        else  if (value <= 10) {
+            System.out.println("Music plays at medium volume");
+        }
+        else if (value <= 15) {
+            System.out.println("The music is playing loudly");
+        } else {
+            System.out.println("The music is playing very loud");
+        }
```
Comment (Evangard) [inline src/Lesson_7/Task_3/MyDevice.java:27]: If you have more than one if/else, then better to use switch/case

Comment (Evangard) [reply src/Lesson_7/Task_3/MyDevice.java:27]: This method should turn on your device, so, you have to check if volume = 0, then you have to turn it on.

File: src/Lesson_7/Task_3/MyDevice.java [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    }
+
+    @Override
+    public void off() {
+        if (value <= 0) {
+            System.out.println("The music stopped playing");
+        } else {
+            System.out.println("Music is playing ");
+        }
+    }
```
Comment (Evangard) [inline src/Lesson_7/Task_3/MyDevice.java:38]: This method should turn off your device, so, you have to check if volume > 0, then you have to turn it off. By the way, volume cannot be < 0. You can add this verification in changeVolume method.

File: src/Lesson_7/Task_2/FahrenheitConverter.java (hunk 1 above)
Comment (OlegShvets1) [reply src/Lesson_7/Task_2/FahrenheitConverter.java:6]: +

File: src/Lesson_7/Task_2/KelvinConverter.java (hunk 2 above)
Comment (OlegShvets1) [reply src/Lesson_7/Task_2/KelvinConverter.java:6]: +

File: src/Lesson_7/Task_3/MyDevice.java (hunk 4 above)
Comment (OlegShvets1) [reply src/Lesson_7/Task_3/MyDevice.java:38]: +

File: src/Lesson_7/Task_3/MyDevice.java (hunk 3 above)
Comment (OlegShvets1) [reply src/Lesson_7/Task_3/MyDevice.java:27]: +

**Review by Evangard — APPROVED** [review]
You can merge it after fixing conflicts.

**Pull request merged**